pkg/downloader/manager.go

```diff
@@ -64,42 +66,102 @@ type Manager struct {
 // If the lockfile is not present, this will run a Manager.Update()
 //
 // If SkipUpdate is set, this will not update the repository.
-func (m *Manager) Build() error {
+func (m *Manager) Build() ([]*chartutil.Dependency, error) {
```

Comment:
This is breaking a publicly exposed API. Anyway you can change this to avoid the breaking change?